gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/jdbc/EmbedStatement.java

```diff
@@ -2064,6 +2064,10 @@ boolean executeStatement(Activation a,
           connForRemote = false;
           tran = null;
         }
+        if (act != null) {
```

Comment:
Yeah again. Why is this required and should we use the timed wait version instead?